Buildings/Controls/OBC/CDL/Logical/Timer.mo

```diff
@@ -3,42 +3,59 @@ block Timer
   "Timer measuring the time from the time instant where the Boolean input became true"
 
   parameter Boolean reset = true
-    "Set to true for reseting timer to zero when input becomes false";
+    "Set to true for reseting timer to zero when input becomes false, set to false when need to find accumulated time which can be reset by another boolean input";
 
   Interfaces.BooleanInput u "Connector of Boolean input signal"
     annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+  Interfaces.BooleanInput u0 if not reset
+    "Connector of Boolean for resetting output to zero"
+    annotation (Placement(transformation(extent={{-140,-90},{-100,-50}}),
+      iconTransformation(extent={{-140,-100},{-100,-60}})));
```

Comment:
The design with the parameter `reset` and an input `u0` is not intuitive. Why not calling the parameter `accumulate` (because it determines whether the timer accumulates time) and calling the input `reset` because it resets the timer when it becomes `true`.
Calling an input `u0` and having the user figure out what the difference between `u` and `u0` is is not user friendly.

Also, why not change the icon to indicate the value of `accumulate`. This would further ease graphical modeling.

Reply:
I like this approach as well.